### ISSUE TITLE
Can not open message action menu using screen reader

### ISSUE THREAD

**Issue by dnz3d4c** [RocketChat/Rocket.Chat#31300]
### Description:
Can not open message action menu using screen reader

### Steps to reproduce:

1. Open any channel and navigate the message area to the screen reader.
2. Use a screen reader to see if there is a button to open the message action menu.

### Expected behavior:
You should be able to recognize and use buttons that can open message action menus using a screen reader.

### Actual behavior:

You can not recognize buttons that can open the message action menu using a screen reader, and you can not use them.
### Server Setup Information:
- Version of Rocket.Chat Server: rocketchat-server 0.69.2
- Operating System: ubuntu 18.04
- Deployment Method: 

Thanks.

**Comment by dougfabris** [comment]
@dnz3d4c It's a critical issue, indeed. In name of the team I'm sorry about the delay to prioritize it. 
Fortunately due some recent efforts, now it's possible to navigate on the message actions as you can check here: #31549
We did a lot of changes regarding the keyboard navigation and a feedback from users would be really appreciated.

Thanks for reporting and helping us!
